### ISSUE TITLE
TV 3D options are grayed out on Philips/Sony Android TV

### ISSUE THREAD

**Issue by gouranga** [koying/SPMC#459]
3D switch and auto-detection doesn't work with SPMC (and Kodi too, tested both 16 and 17 branches) set up directly on TV — the options in the menu are grayed out. Everything else works as expected.

My TV is Philips 49PUS7100/60, but as far as get it doesn't work on any [Philips](http://www.supportforum.philips.com/en/showthread.php?18882-Using-Kodi-no-3D-format-options-are-available) nor [Sony](https://community.sony.co.uk/t5/televisions/2015-android-tv-bugs-and-issues/m-p/2126443#M41214) Android TV:

![2016-09-14 11 04 35](https://cloud.githubusercontent.com/assets/1222966/18504526/6fad99ee-7a6b-11e6-85ff-7714cec14316.jpg)

I'm not really sure if it's related to the SPMC itself, but TV vendors are silent about this issue.

**Comment by koying** [comment]
On Sony, you can't get 3D in 4K, and you can't force the TV to 1080p, so won't work, currently.
I imagine it's the same on Philips...

**Comment by gouranga** [comment]
On Philips anything related to Android is 1080p, no 4k in apps.

**Comment by koying** [comment]
Indeed, the android subsystem is stuck in 1080p, but the "TV" subsystem, which greys out the 3D options, is in 4K.

Apparently, the 2 subsystems are isolated, which prevents to switch the TV in 1080p from android.
